docs/ABOUT.md

```diff
@@ -1,16 +1,9 @@
-PureScript is a purely functional, statically-typed programming language with
-type inference.
+PureScript is a purely functional, statically-typed programming language with global type inference.
 
-**Functional** means that functions are first-class data types.
+**Functional** means computation is modeled as the evaluation of expressions rather than the execution of a sequence of instructions.
```

Comment:
Hmm. You're right. Maybe we can say something like 

> **Functional** means that functions are first-class data types. Functions are an important and powerful tool for abstraction. In PureScript, computation is modeled as the evaluation of expressions made up of function applications, rather than as the execution of a sequence of instructions.

What do you think?